packages/core-ethereum/crates/core-ethereum-misc/Cargo.toml

```diff
@@ -13,19 +13,35 @@ crate-type = ["cdylib", "rlib"]
 
 [features]
 default = ["console_error_panic_hook", "wasm"]
-wasm = [ "dep:wasm-bindgen" ]
+wasm = [ "dep:wasm-bindgen", "dep:js-sys" ]
 console_error_panic_hook = [ "dep:console_error_panic_hook" ]
 
 [dependencies]
-wasm-bindgen = { workspace = true, optional = true }
+async-std = { version = "1.12.0", features = ["attributes"] }
+async-trait = "0.1"
+core-crypto = { path = "../../../core/crates/core-crypto" }
+core-ethereum-db = { path = "../core-ethereum-db" }
+futures = "0.3"
+js-sys = { version = "0.3.63", optional = true }
+libp2p-identity = "0.1.2"
+thiserror = "1.0"
+utils-db = { path = "../../../utils/crates/utils-db" }
+utils-log = { path = "../../../utils/crates/utils-log" }
 utils-proc-macros = { path = "../../../utils/crates/utils-proc-macros" }
+utils-types = { path = "../../../utils/crates/utils-types" }
+wasm-bindgen = { workspace = true, optional = true }
 
 console_error_panic_hook = { version = "0.1.7", optional = true }
 # wee_alloc = { version = "0.4.5", optional = true }
 
 [dev-dependencies]
+mockall = "0.11.4"
+hex-literal = "0.4"
 wasm-bindgen-test = "0.3.36"
 
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+rusty-leveldb = "1.0"
```

Comment:
Why is this needed, when the core-ethereum-db is included?

Reply:
Right, good catch.

Reply:
It is still needed, because CoreEthereum-db only has the adapter (shim), but the actual Rusty-LevelDB backend class is in the `rusty-levedb` crate.